crates/primitives/receipt/src/from_blockifier.rs

```diff
@@ -113,16 +143,15 @@ pub fn from_blockifier_execution_info(res: &TransactionExecutionInfo, tx: &Trans
                 execution_result,
             })
         }
-        Transaction::L1HandlerTransaction(_tx) => unimplemented!("l1 handler tx"),
-        // Transaction::L1HandlerTransactionv(tx) => TransactionReceipt::L1Handler(L1HandlerTransactionReceipt {
-        //     transaction_hash,
-        //     actual_fee,
-        //     messages_sent,
-        //     events,
-        //     execution_resources,
-        //     execution_result,
-        //     message_hash: todo!(),
-        // }),
+        Transaction::L1HandlerTransaction(_tx) => TransactionReceipt::L1Handler(L1HandlerTransactionReceipt {
+            transaction_hash,
+            actual_fee,
+            messages_sent,
+            events,
+            execution_resources,
+            execution_result,
+            message_hash: Felt::from(1),
```

Comment:
add a TODO?

Reply:
i dont remember how the message hash is computed, i think the stuff that's commented just on top may work?
we should make a unit test for that tbh

Reply:
it's implemented now